src/sentry/models/groupsubscription.py

```diff
@@ -20,6 +20,7 @@ class GroupSubscriptionReason(object):
     assigned = 2
     bookmark = 3
     status_change = 4
+    setting = 5
```

Comment:
yeah definitely, thanks. What are your thoughts on UserOrgOption and UserOption here?